src/helpers/collect-linked-pulls.ts

```diff
@@ -34,6 +34,7 @@ const query = /* GraphQL */ `
                   id: databaseId
                 }
               }
+              reviewDecision
```

Comment:
I guess this is based on the last review?

Reply:
According to the docs it contains the last global state of the review: https://docs.github.com/en/graphql/reference/enums#pullrequestreviewdecision